### ISSUE TITLE
[BUG] <My lumberjack refuses to cut birch wood>

### ISSUE THREAD

**Issue by Valkrane** [ldtteam/minecolonies#9613]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [ ] Multi Player

### Minecraft Version

1.19.2

### MineColonies Version

minecolonies-1.20.1-1.1.350-BETA

### Structurize Version

structurize-1.20.1-1.0.662-BETA

### Related Mods and their Versions

![image](https://github.com/ldtteam/minecolonies/assets/153466055/0d4aee11-ca74-4c65-b5eb-dc0380fcb8bb)

### Current Behavior

No matter how I define a zone for him, put him in the meadow biome and plant birch trees all around him, he runs over the cherry trees. I have no solution, please help me!

### Expected Behavior

Je voudrais que mon bucheron, coupe du bois de bouleau

### Reproduction Steps

Vous mettre dans une prairie.
Planter des bouleaux
Créer une hutte de bucheron.
Constater qu'il ne coupe pas de bouleau. 

### Logs

<script src="https://gist.github.com/Valkrane/2ce8d9fee4f8f0627f56cf6144902890.js"></script>

### Anything else?

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Moobien** [comment]
do you have birch saplings enabled? do you have a lot of different tree types in the forester's area?

**Comment by Pchelekk** [comment]
Yep, same issue. He also is ignoring Oak Trees, not sure about other types, he is only cutting Spruce Trees for me (I have a farm with only Spruce, Oak and Birch Trees). Also not sure if it is a glitch but whenever the tree is 1 block above the default surface level, he ignores the Spruce Tree aswell. I have a ground and I placed 1 dirt on top of the ground and on top of that a Spruce Tree, he is ignoring that aswell, but when the Spruce Tree is at ground level, the Tree gets cut. 
![2023-12-12_15 11 33](https://github.com/ldtteam/minecolonies/assets/153627129/3f421c7e-7d1d-49ee-a653-3b78eb36c41f)

**Comment by Raycoms** [comment]
Yes, blocks on dirt are ignored, pathfinder cant find those blocks.

**Comment by Pchelekk** [comment]
Ok, then he is ignoring oak and birch trees on my side. (on default ground level)

**Comment by MotionlessTrain** [comment]
Are those planted inside of the colony, inside of the zone (if set), or if no zone is set, outside of all buildings?

**Comment by Pchelekk** [comment]
Before they were planted next to the Forester's Hut inside the colony range but only Spruce Trees were being cut off, so I created a zone using the Selection Tool inside the Forester's Hut GUI. Now They are inside the zone that I created (X,Y,Z are big enough so they aren't exiting the zone) but still only Spruce Trees are getting Cut by the Forester. Yes, the trees aren't in a Hitbox of a Building's range. (Also all of the Saplings in The Forester's Hut are turned to on).
